src/plots/cartesian/layout_attributes.js

```diff
@@ -248,6 +248,135 @@ module.exports = {
             'Moreover, note that matching axes must have the same `type`.'
         ].join(' ')
     },
+
+    breaks: templatedArray('break', {
+        enabled: {
+            valType: 'boolean',
+            role: 'info',
+            dflt: true,
+            editType: 'calc',
+            description: [
+                'Determines whether this axis break is enabled or disabled.'
+            ].join(' ')
+        },
+
+        bounds: {
+            valType: 'info_array',
+            role: 'info',
+            items: [
+                {valType: 'any', editType: 'calc'},
+                {valType: 'any', editType: 'calc'}
+            ],
+            editType: 'calc',
+            description: [
+                'Sets the lower and upper bounds of this axis break.',
+                'Can be used with `operation` to determine the behavior at the bounds.',
+                'On *date* axes, it can be used `pattern`.'
```

Comment:
missing "with" before "pattern" ?

Reply:
Good eye! Fixed in https://github.com/plotly/plotly.js/pull/4614/commits/76a265e75e7f6f67169710ec292b625ef9b8e172